eng/Version.Details.xml

```diff
@@ -4,9 +4,9 @@
       <Uri>https://github.com/dotnet/standard</Uri>
       <Sha>cfe95a23647c7de1fe1a349343115bd7720d6949</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Runtime.ICU.Transport" Version="5.0.0-preview.8.20410.1">
+    <Dependency Name="Microsoft.NETCore.Runtime.ICU.Transport" Version="6.0.0-alpha.1.20425.2">
```

Comment:
Why 6.* ?

Reply:
@akoeplinger any idea? it's the latest version of ICU in dotnet5 feed. and I don't see any "6.0" string in the repo

Reply:
@EgorBo that's the wrong package version, you want this one: https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet5-transport&package=Microsoft.NETCore.Runtime.ICU.Transport&protocolType=NuGet&version=5.0.0-rc.1.20425.3&view=overview